packages/itwinui-variables/index.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './mixins';
+
+:root {
+  @include light-variables;
+
+  @media (prefers-color-scheme: dark) {
+    @include dark-variables;
+  }
+
+  @media (prefers-color-scheme: light) and (prefers-contrast: more) {
+    @include light-hc-variables;
+  }
+
+  @media (prefers-color-scheme: dark) and (prefers-contrast: more) {
+    @include dark-hc-variables;
+  }
+}
+
+[data-iui-theme='light'] {
```

Comment:
So we would end up with a single set of variables included right? 

First goes default OS and then override with attribute value (if it exists)

Reply:
yeah, that's if the data attribute is applied on the `<html>` element.

if it's applied on some bespoke element deeper in the DOM, it will override the one from `:root`, as intended.